src/Bridge/Symfony/Routing/Router.php

```diff
@@ -71,7 +71,7 @@ public function match($pathInfo)
         $baseContext = $this->router->getContext();
         $pathInfo = str_replace($baseContext->getBaseUrl(), '', $pathInfo);
 
-        $request = Request::create($pathInfo);
+        $request = Request::create($pathInfo, 'GET', [], [], [], ['HTTP_HOST' => $baseContext->getHost()]);
```

Comment:
You can just call `$context->setHost($baseContext->getHost());` instead of this change. It will be a bit cleaner.